README.md

```diff
@@ -0,0 +1,89 @@
+<div align="center">
+  <br/>
+  <a href="https://github.com/zaproxy/zap-api-nodejs" title="zaproxy">
+    <img width=186px src="assets/images/ZapNodeApi.png" alt="zap-api-nodejs logo">
+  </a>
+  <br/>
+<br/>
+<h2>OWASP ZAP Node API</h2>
+<br/><br/>
+
+<a href="https://www.apache.org/licenses/LICENSE-2.0.html" title="License">
+  <img src="https://img.shields.io/badge/license-Apache%202-4EB1BA.svg" alt="License"/>
+</a>
+
+<br/><br/>
+</div>
+
+The NodeJS implementation to access the [OWASP ZAP API](https://github.com/zaproxy/zaproxy/wiki/ApiDetails). For more information
+about OWASP ZAP consult the (main) [OWASP ZAP project](https://github.com/zaproxy/zaproxy/).
+
+## Install
+
+```
+npm install zaproxy
+```
+
+## Usage
+
+By default Zap [requires an API key](https://github.com/zaproxy/zaproxy/wiki/FAQapikey) to be sent with every request. This is done automatically providing you supply the same API key when you instantiate the `ZapClient` that you use to run Zap with. All following API requests will use this same API key.  
+You can dissable the API key when running Zap if you are on a trusted network and understand the risks. If you choose to do so, you may omit the `apiKey` property of the `zapOptions` object supplied to `ZapClient`.
+
+### Instantiate the client API:
+
+```js
+const ZapClient = require('zaproxy');
+
+const zapOptions = {
+  apiKey: <the key you supplied to Zap when you started it>, // I.E. 'v90dnblpqs1pcac991tn2oudl'
+  proxy: <protocol>://<host>:<port> // I.E. 'http://192.168.0.10:8080'
+};
+
+const zaproxy = new ZapClient(zapOptions);
+```
+
+### Use the client API:
+
+Callbacks:
+
+```js
+zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren, recurse, subtreeonly, (err, resp) => {
+  if (err) // Handle the error.
+  if (resp) // Handle the response.
+});
+```
+
+Promises:
+
+```js
+await zaproxy.spider.scanAsUser(contextId, userId, sutBaseUrl, maxChildren)
+  .then(
+    resp => console.log(JSON.stringify(resp)),
+    err => `Error occured while attempting to scan as user. Error was: ${err.message}`
+  );
+```
+
+## API
+
+For a full API list, see [https://github.com/zaproxy/zaproxy/wiki/ApiGen_Index](https://github.com/zaproxy/zaproxy/wiki/ApiGen_Index).
+
+The Node.js API methods have the same signature as the API documentation, Featuring both callback and promise based interfaces, making everyone happy.
```

Comment:
Should be a dot before `Featuring` (or should be lower case)?

Reply:
Fixed.